catalog-info.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: autodev-action
+  description: This GitHub action merges labeled PRs into the dev branch.
+  annotations:
+    github.com/project-slug: Staffbase/autodev-action
+  tags:
+    - github-action
+    - typescript
+spec:
+  type: other
+  lifecycle: production
```

Please add missing new-line to end of file

```suggestion
  lifecycle: production

```